src/rookify/modules/__init__.py

```diff
@@ -55,6 +56,20 @@ def load_modules(machine: Machine, config: Dict[str, Any]) -> None:
     :return: returns tuple of preflight_modules, modules
     """
 
+    migration_modules = config["migration_modules"].copy()
+
     for entry in importlib.resources.files("rookify.modules").iterdir():
         if entry.is_dir() and entry.name in config["migration_modules"]:
+            migration_modules.remove(entry.name)
             _load_module(machine, config, entry.name)
+
+    if len(migration_modules) > 0 or len(config["migration_modules"]) < 1:
+        logger = get_logger()
+
+        if len(config["migration_modules"]) < 1:
```

Comment:
Im not sure if this situation will be able occure,  but I dont think it hurts :)